### PR TITLE
fix: does run instrumented function leak promises

### PR DESCRIPTION
**Pull request by pauldambra**
blob ingestion runs hot on RAM
when load reduces RAM does not reduce

this blob suggests that returning but not dealing with promises can cause this problem https://jakearchibald.com/2024/garbage-collection-and-closures/

i've not looked at the speed graphs for a long time because they tend to confirm that JS parsing is the expensive bit

let's remove `runInstrumentedFunction` to rule in or out that this is happening here

### CODE REVIEW

File: plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts [hunk 1]
```diff
@@ -360,91 +359,54 @@ export class SessionRecordingIngester {
             })
         }
 
-        await runInstrumentedFunction({
```
Comment (pauldambra) [inline plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts:363]: so my theory/guess/wondering is that `runInstrumentedFunction` is returning a promise which we asait but don't deal with

and so the promise hangs around like an unwanted guest at a funeral causing the high RAM

Comment (pauldambra) [reply plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts:363]: this of course might have no impact at all on blobby's RAM usage

**Review by benjackwhite — APPROVED** [review]
I don't at all understand how this could be an issue... The promises aren't dangling if they are awaited right? The article about promises eating memory I thought was only applying in the held state which these wouldnt be... In any case fine to test it but maybe good to revert if it makes no difference as these metrics were sometimes pretty useful.

**Comment by pauldambra** [comment]
> I don't at all understand how this could be an issue... 

@benjackwhite yep, totally clutching at straws but cheap to test!

**Pull request merged**